### ISSUE TITLE
Documentation fixes

### ISSUE THREAD

**Issue by piskvorky** [RaRe-Technologies/gensim#2037]
This issue collects PRs related to improving the Gensim documentation.

Merged
--------

https://github.com/RaRe-Technologies/gensim/pull/1633
https://github.com/RaRe-Technologies/gensim/pull/1625
https://github.com/RaRe-Technologies/gensim/pull/1640
https://github.com/RaRe-Technologies/gensim/pull/1702
https://github.com/RaRe-Technologies/gensim/pull/1684
https://github.com/RaRe-Technologies/gensim/pull/1709
https://github.com/RaRe-Technologies/gensim/pull/1739
https://github.com/RaRe-Technologies/gensim/pull/1681
https://github.com/RaRe-Technologies/gensim/pull/1806
https://github.com/RaRe-Technologies/gensim/pull/1802
https://github.com/RaRe-Technologies/gensim/pull/1797
https://github.com/RaRe-Technologies/gensim/pull/1804
https://github.com/RaRe-Technologies/gensim/pull/1803
https://github.com/RaRe-Technologies/gensim/pull/1805
https://github.com/RaRe-Technologies/gensim/pull/1714
https://github.com/RaRe-Technologies/gensim/pull/1814
https://github.com/RaRe-Technologies/gensim/pull/1729
https://github.com/RaRe-Technologies/gensim/pull/1793
https://github.com/RaRe-Technologies/gensim/pull/1801
https://github.com/RaRe-Technologies/gensim/pull/1913
https://github.com/RaRe-Technologies/gensim/pull/1859
https://github.com/RaRe-Technologies/gensim/pull/1835
https://github.com/RaRe-Technologies/gensim/pull/1792
https://github.com/RaRe-Technologies/gensim/pull/1904
https://github.com/RaRe-Technologies/gensim/pull/1910
https://github.com/RaRe-Technologies/gensim/pull/1919
https://github.com/RaRe-Technologies/gensim/pull/1892
https://github.com/RaRe-Technologies/gensim/pull/1880
https://github.com/RaRe-Technologies/gensim/pull/1861
https://github.com/RaRe-Technologies/gensim/pull/1876

WIP
----

https://github.com/RaRe-Technologies/gensim/pull/2026
https://github.com/RaRe-Technologies/gensim/pull/1809
https://github.com/RaRe-Technologies/gensim/pull/1944

**Comment by Laubeee** [comment]
It seems some parameters mismatch what is implemented vs in the docs. e.g.
https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/models/ldamodel.py#L372
i think 'default' should be 'auto'?

**Comment by menshikh-iv** [comment]
@Laubeee yes, you are right, good catch! Can you submit PR with the fix, please?

**Comment by Laubeee** [comment]
here you go: #2156
